Dockerfile

```
@@ -0,0 +1,11 @@
+FROM php:7.3-fpm
+
+RUN apt-get update && apt-get install -y libmcrypt-dev \
+  mariadb-client libmagickwand-dev --no-install-recommends \
+  && pecl install imagick \
+  && docker-php-ext-enable imagick \
+  && docker-php-ext-install pdo_mysql
+
```

Add user ownership to `/var/www` to avoid permission error on Linux systems
```suggestion
RUN chown -R www-data:www-data /var/www
```